src/NUnitFramework/framework/Api/DefaultTestAssemblyBuilder.cs

```diff
@@ -49,6 +49,11 @@ public class DefaultTestAssemblyBuilder : ITestAssemblyBuilder
         /// </summary>
         readonly ISuiteBuilder _defaultSuiteBuilder;
 
+        /// <summary>
+        /// The PreFilter used by this builder
```

Comment:
It doesn't like like the XML comment adds any info beyond what you see by looking at the field name.

Reply:
Well, yes. But it's required. 😸 What would you like it to say?

Reply:
It looks like it should not be required because it's on a private field. ❔